### ISSUE TITLE
getting error msg while changing Tier in Storage explorer

### ISSUE THREAD

**Issue by srikrishna1972** [microsoft/AzureStorageExplorer#8382]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Storage Explorer Build Number

20241029.2

### Platform

All

### OS Version

Windows 11 Enterprise

### Bug Description

getting error msg while changing Tier in Storage explorer

### Steps to Reproduce

1 Launch storage explorer
2. go to Blob which needs to be Tier changed from Archive to Hot.
3. select standard/premium to run the process.
4. collect the log after completion of run from status.
5. click on details on right side of status section.

### Actual Experience

Failures occur while changing access tier in container '*****![Image](https://github.com/user-attachments/assets/52fe4122-715a-46d1-a997-90ff0cbdc8c4)
' to 'Hot': success: 14645, failure: 37

### Expected Experience

Expected should be all success full.

**Comment by craxal** [comment]
@srikrishna1972 Can you try the operation again on one or more of the failed bobs? If the problem persists, are you able to successfully change the access tier from the Portal?

**Comment by srikrishna1972** [comment]
I have tried again and even though errors shows some failed but I do not see the tier in archive . Everything got changed tier.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by craxal** [comment]
@srikrishna1972 So, are you saying that, even though you see several errors, all of your blobs successfully transitioned from Archive to Hot?

**Comment by srikrishna1972** [comment]
Yes Craig I do not see any blobs and files which are in archive.

**Comment by craxal** [comment]
@srikrishna1972 Can you reproduce the error and provide your app logs here for us to look at?

**Comment by craxal** [comment]
Closing due to inactivity. If you are still having problems after answering the above questions, please leave a comment, and we can revisit.
